### DISCUSSION TITLE
ECTyper Galaxy tool update!

### DISCUSSION THREAD

**Discussion by JayanthiGangiredla** [phac-nml/ecoli_serotyping#88] [General]
Hi all,

Thank you for maintaining ectyper on galaxy. I was wondering how often do you update the database for galaxy users? This tool has been used a lot for e.coli research and wondering if you can update the newer version of tool with an updated database. the last update was in 2020. Thanks again!

Thanks,
Jayanthi

**Comment by kbessonov1984** [comment]
Thank you for reaching out. The galaxy tool is just interface to the command line ECTYPER tool. The database of the command line tool is in the JSON format (https://github.com/phac-nml/ecoli_serotyping/blob/master/ectyper/Data/ectyper_alleles_db.json) and can be easily manually updated. We have not updated the database of antigens yet as E.coli nomenclature is fairly stable and we did not see a pressing need to include a new antigen in our routine surveillance.  There are some temporary O-antigens designated as OX groups  and subtypes, but those were not included in the database for practical reasons. In future we are looking to do database revisions and include new subtypes and new established antigens. If you have suggestion to include some new sequences and antigens then we can look at solutions (e.g. allow users to easily add their sequences or add them permanently to the database).  Looking forward to your suggestions and input.
